package.json

```diff
@@ -37,6 +37,7 @@
     "package.json": "pnpm sort-package-json"
   },
   "devDependencies": {
+    "@ark/attest": "0.9.2",
```

Comment:
I saw you did this in another PR and I had moved it out of the workspace root in a follow up PR to be consistent with how we've historically managed deps. Open to changing this approach though and curious to hear your thoughts - should common test deps be added to the workspace root instead of individual packages?

Reply:
Yes, devDependencies should generally be managed at the monorepo root in my opinion. It prevents having to manage multiple versions and continue to install them in new packages as needed, and there's really no downside as long as you're not importing them in source. I have an eslint rule that warns if I'm importing from a package not listed in dependencies.

Reply:
cool, added a follow up here: https://github.com/latticexyz/mud/issues/2962